### PR TITLE
Adequacy patch CSR - revamp output variables [ANT-1932]

### PR DESCRIPTION
**Pull request by flomnes**
### Variables
- Remove Spilled CSR
- Add DTG MRG CSR
- Add UNSP. ENRG CSR

### Input parameters
Remove flag "recompute-dtg-mrg"

Make code mode testable by using functions.

TODO
- [x] Add unit tests
- [x] Check code associated to districts
- [x] Update documentation

### CODE REVIEW

File: src/solver/optimisation/adequacy_patch_csr/post_processing.cpp [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+*/
+#include "antares/solver/optimisation/adequacy_patch_csr/post_processing.h"
+
+#include <cmath>
+
+namespace Antares::Data::AdequacyPatch
+{
+double recomputeDTG_MRG(bool trigerred, double dtgMrg, double ens)
+{
+    if (trigerred)
```
Comment (a-zakir) [inline src/solver/optimisation/adequacy_patch_csr/post_processing.cpp]:
```suggestion
double recomputeDTG_MRG(bool triggered, double dtgMrg, double ens)
{
    if (triggered)
```

File: .github/workflows/ubuntu.yml [hunk 2]
```diff
@@ -25,8 +25,8 @@ on:
 env:
   GITHUB_TOKEN: ${{ github.token }}
   IS_RELEASE: ${{ github.event_name == 'workflow_dispatch' }}
-  RUN_SIMPLE_TESTS: ${{ github.event_name == 'push' || inputs.run-tests == 'true' }}
-  RUN_EXTENDED_TESTS: ${{ github.event_name == 'schedule' || inputs.run-tests == 'true' }}
+  RUN_SIMPLE_TESTS: false
```
Comment (payetvin) [inline .github/workflows/ubuntu.yml]: revert

File: .github/workflows/windows-vcpkg.yml [hunk 3]
```diff
@@ -25,8 +25,8 @@ on:
 env:
   GITHUB_TOKEN: ${{ github.token }}
   IS_RELEASE: ${{ github.event_name == 'workflow_dispatch' }}
-  RUN_SIMPLE_TESTS: ${{ github.event_name == 'push' || inputs.run-tests == 'true' }}
-  RUN_EXTENDED_TESTS: ${{ github.event_name == 'schedule' || inputs.run-tests == 'true' }}
+  RUN_SIMPLE_TESTS: false
```
Comment (payetvin) [inline .github/workflows/windows-vcpkg.yml]: revert

**Review by payetvin — CHANGES_REQUESTED** [review]
Documentation isn't complete

**Review by payetvin — APPROVED** [review]

**Pull request merged**